flake.nix

```diff
@@ -40,6 +40,28 @@
         # From https://nixos.wiki/wiki/Google_Cloud_SDK
         gdk = pkgs.google-cloud-sdk.withExtraComponents ([ pkgs.google-cloud-sdk.components.gke-gcloud-auth-plugin ]);
 
+        customProtocGenGo = pkgs.buildGoModule rec {
+          name = "protoc-gen-go";
+          owner = "protocolbuffers";
+          repo = "protobuf-go";
+          rev = "v1.30.0"; 
```

Comment:
Hhmm I'm not seeing this version in the list?
https://pkg.go.dev/github.com/golang/protobuf/protoc-gen-go?tab=versions